src/arrays/referencing_array_and_its_elements/medium/q9/EveryEven.java

```diff
@@ -0,0 +1,35 @@
+package arrays.referencing_array_and_its_elements.medium.q9;
+
+/*
+Write a Java program that initializes an array of 10 integers randomly generated between 1 and 1000 (inclusive on both ends)
```

Comment:
Change to:
"Write a Java program that initializes an array of 1000 randomly generated integers between 1 and 1000 (inclusive on both ends) and then prints the array. Then initialize a second array to be half the size of the first and copy all values at even indexes from the first array and store them in the second array starting at 0. Then print out the second array."

Reply:
Change question so the first array size is 1000 to force them to make an array